index.js

```diff
@@ -158,6 +185,38 @@ function generatePowerGeneration (geo, subIndicators) {
   }
 }
 
+// Generate data for the Clean Energy Investment Chart for a single country
+function generateEnergyInvestment (geo, investments) {
+  return {
+    'id': 'cleanEnergyInvestment',
+    'meta': {
+      'label-y': 'usd/m',
+      'label-x': 'year',
+      'title': 'Clean Energy Investment'
+    },
+    'data': investments
+      .filter(i => i.geography === geo.name)
+      .reduce((a, b) => {
```

Comment:
It is a better practice to use `acc` for the accumulator as it makes it clearer what we're dealing with. Using `a` and `b` seems like you're working with similar objects.